### DISCUSSION TITLE
Is it safe to store document scans, like passports, social security numbers and etc. on GitHub private repo?

### DISCUSSION THREAD

**Discussion by arudaev** [community/community#33876] [New to GitHub]
I am not a fan of paying for DropBox and Evernote to store essential files. How safe is GitHub private repo?

**Comment by radar-blip** [comment]
It should be safe, but theres a huge privacy concern/problem on the internet nowdays. 
I dont think I would put stuff like that online anywhere, unless absolutely necessary.

**Reply by arudaev** [reply]
I mean... If you were to lose the original document, or go to a different country (for a vacation). Isn't it better to have an online copy instead of a physical one?

**Reply by radar-blip** [reply]
If that were the case, then yes. If its necessary its necessary.
